warehouse/warehouse.go

```diff
@@ -453,13 +462,18 @@ func (wh *HandleT) getPendingUploads(warehouse warehouseutils.WarehouseT) ([]Upl
 	return uploads, nil
 }
 
+func namespaceIdentifier(warehouse warehouseutils.WarehouseT) string {
```

Comment:
Why not use workerIdentifier itself instead?

Reply:
@sivashanmukh missed that the func exists, used that